tools/src/wyvern/target/corewyvernIL/ASTNode.java

```diff
@@ -18,8 +20,8 @@
 	/* TODO: eventually get rid of this constructor if we can,
 	 * so that every ASTNode has a valid FileLocation */
 	public ASTNode() {
-		location = null;
-    metadataSet = new HashSet<>();
+      this((FileLocation)null);
+      location = null;
```

Comment:
This duplicates the initialization of location in the other constructor that's called in the previous line